client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,673 @@
+	'School project': i18n.translate( 'School project' ),
+	'School website': i18n.translate( 'School website' ),
+	Science: i18n.translate( 'Science' ),
+	Scouts: i18n.translate( 'Scouts' ),
+	Security: i18n.translate( 'Security' ),
+	'Self care': i18n.translate( 'Self care' ),
+	'Self improvement': i18n.translate( 'Self improvement' ),
+	'Self improvement services': i18n.translate( 'Self improvement services' ),
+	'Self-help': i18n.translate( 'Self-help' ),
+	'Selling art': i18n.translate( 'Selling art' ),
+	'Selling clothes': i18n.translate( 'Selling clothes' ),
+	'Selling products': i18n.translate( 'Selling products' ),
+	Seniors: i18n.translate( 'Seniors' ),
+	'Seo services': i18n.translate( 'Seo services' ),
+	'Service agency': i18n.translate( 'Service agency' ),
+	Sewing: i18n.translate( 'Sewing' ),
+	Shamanism: i18n.translate( 'Shamanism' ),
+	Shirts: i18n.translate( 'Shirts' ),
+	Shoes: i18n.translate( 'Shoes' ),
+	Shopping: i18n.translate( 'Shopping' ),
+	'Short stories': i18n.translate( 'Short stories' ),
+	Singing: i18n.translate( 'Singing' ),
+	'Singing lessons': i18n.translate( 'Singing lessons' ),
+	Skincare: i18n.translate( 'Skincare' ),
+	Smoothies: i18n.translate( 'Smoothies' ),
+	Soccer: i18n.translate( 'Soccer' ),
+	'Social media': i18n.translate( 'Social media' ),
+	Software: i18n.translate( 'Software' ),
+	'Software and hardware': i18n.translate( 'Software and hardware' ),
+	Solutions: i18n.translate( 'Solutions' ),
+	Songs: i18n.translate( 'Songs' ),
+	Space: i18n.translate( 'Space' ),
+	'Speed dating': i18n.translate( 'Speed dating' ),
+	Spiritual: i18n.translate( 'Spiritual' ),
+	'Spiritual coaching': i18n.translate( 'Spiritual coaching' ),
+	Spirituality: i18n.translate( 'Spirituality' ),
+	Sport: i18n.translate( 'Sport' ),
+	Sports: i18n.translate( 'Sports' ),
+	'Stationary supplies': i18n.translate( 'Stationary supplies' ),
+	Statistics: i18n.translate( 'Statistics' ),
+	'Steel protection': i18n.translate( 'Steel protection' ),
+	'Stock market': i18n.translate( 'Stock market' ),
+	Stocks: i18n.translate( 'Stocks' ),
+	Store: i18n.translate( 'Store' ),
+	Storefront: i18n.translate( 'Storefront' ),
+	Stories: i18n.translate( 'Stories' ),
+	Student: i18n.translate( 'Student' ),
+	'Student association': i18n.translate( 'Student association' ),
+	'Student housing': i18n.translate( 'Student housing' ),
+	'Students union': i18n.translate( 'Students union' ),
+	Style: i18n.translate( 'Style' ),
+	Stylist: i18n.translate( 'Stylist' ),
+	Subscriptions: i18n.translate( 'Subscriptions' ),
+	'Suicide prevention': i18n.translate( 'Suicide prevention' ),
+	Supplements: i18n.translate( 'Supplements' ),
+	Supplies: i18n.translate( 'Supplies' ),
+	Support: i18n.translate( 'Support' ),
+	'Sustainable development': i18n.translate( 'Sustainable development' ),
+	'Swimming pools': i18n.translate( 'Swimming pools' ),
+	'Swimming pools repairs': i18n.translate( 'Swimming pools repairs' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Swimmping pools repairs' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).